Modules/posixmodule.c

```diff
@@ -13815,7 +13829,9 @@ os_cpu_count_impl(PyObject *module)
 {
     int ncpu = 0;
 #ifdef MS_WINDOWS
+#ifndef MS_WINDOWS_NON_DESKTOP
```

Comment:
What's the point of having `os.cpu_count()` that always returns 0? The definition of `os_cpu_count_impl()` should be skipped for non-desktop builds.

Reply:
I did this, since this way it returns None: https://github.com/python/cpython/blob/2ba0a87c338a5a4b629d51f5ae3276a134728bc4/Modules/posixmodule.c#L13853-L13856

According to the documentation the function returns None if the count is indeterminable

Reply:
Of course, you're right.